MetaMask/metamask-extension
# UX: Send Flow: Implement Tabbed Interface for Accounts + Contacts

## **Description**

We decided to implement a tabbed interface for the new send flow with regard to the address book and accounts.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1636

## **Manual testing steps**

1. Open the new send flow
2. See contacts listed under the Contacts tab
3. See accounts under the accounts tab

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="399" alt="SCR-20231107-mtsk" src="https://github.com/MetaMask/metamask-extension/assets/46655/73137494-cd5b-4e4a-90f6-e975b85c8570">
<img width="380" alt="SCR-20231107-mrue" src="https://github.com/MetaMask/metamask-extension/assets/46655/2164912a-b6f2-4dc4-9d6a-2aca76a74ed9">
<img width="408" alt="SCR-20231107-mrsw" src="https://github.com/MetaMask/metamask-extension/assets/46655/532b5215-cff0-4713-a537-9ea7f7daa56d">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
